pkg/virt-launcher/notify-client/client.go

```diff
@@ -201,7 +201,6 @@ func eventCallback(c cli.Connection, domain *api.Domain, libvirtEvent libvirtEve
 	if err != nil {
 		if !domainerrors.IsNotFound(err) {
 			log.Log.Reason(err).Error("Could not fetch the Domain.")
-			client.SendDomainEvent(newWatchEventError(err))
```

Comment:
I think one thing to consider is, if libvirtd gets e.g. killed an restarted, if we will now still see an update of the latest domain state being transferred over to virt-handler. Is that the case?

Reply:
i think this one is ?
https://github.com/kubevirt/kubevirt/pull/3806/files#diff-7175e08324a35a860b12d6518d396988L339
and if so, it will be the case that ERROR would be sent, as David said (hopefully that i understood right)